_data/header/header.yml

```diff
@@ -1,5 +1,3 @@
-tagline: The Future of Complex Rehab
-popoutTagline: Building Advanced Tools for ATP's
-popoutTextBlock: >-
-  CRS is dedicated to improving the productivity and efficiency of the Assistive
-  Technology Professional (ATP).
```

Comment:
Since you removed these variables (tagline, popoutTagline, popoutTextBlock) -- can you also remove them from the config.yml s for the CMS?